### ISSUE TITLE
Incomplete instruction of local setup

### ISSUE THREAD

**Issue by jcguu95** [fniessen/org-html-themes#94]
Thanks for the amazing work! I'd like to point out one little issue though.

Currently, the instruction for setting up the theme locally in the `readme` file is not complete. I had the impression (and I believe this is all what it says) that it suffices to clone the repo, put the following in the beginning of my org file
```
#+SETUPFILE: path/to/Git/repo/setup/THEME-NAME-local.setup
```
, and simply runs `M-x org-html-export-to-html`. This did not, however, render me the right file. One also has to do this in a directory where the `styles` folder lies.

I understand this is just a minor issue, and can be fixed by most users if they pay more attention to what the `.setup` files say, but this could be scary for new/impatient users. There are several other similar packages outside, and I believe this minor change will make this package much more competitive.

This post is related to this [issue](https://github.com/fniessen/org-html-themes/issues/90).

**Comment by fniessen** [comment]
@jcguu95 , thanks for your comments. Well, yes, I believe it's not perfect yet, well heading to it ;-)

Would you mind completing the instructions, or adapting them, in order to make this easier for newcomers?

**Comment by hasan-sh** [comment]
@fniessen is there any way to have `#+SETUPFILE: THEME_URL` as a default setup for all org files?

**Comment by fniessen** [comment]
@hasan-sh , not that I know, no. But that could be a good suggestion for the Org-mailing list...

**Comment by hasan-sh** [comment]
Yes I'd say so. I'm not sure about this but is it possible to add a theme in the `org-export-insert-default-template`; `C-c C-e #`!?

**Comment by hasan-sh** [comment]
Please excuse my questions if they seem trivial, but I've just started using (spac)emacs!

**Comment by fniessen** [comment]
No problem, that should be editable indeed.  At least, I would expect it. But I did not find the right variable for doing so.

Please address this question to the Org team.
